libbeat/tests/compose/compose.go

```diff
@@ -80,17 +80,17 @@ func EnsureUp(t testing.TB, service string, options ...UpOption) HostInfo {
 		}
 
 		// Wait for health
-		err = compose.Wait(upOptions.Timeout, service)
```

Comment:
I would suggest uncommenting this, and we can still wait for the health check to complete.